pkg/manager/manager.go

```diff
@@ -164,6 +166,17 @@ type Options struct {
 	// It can be set to "0" to disable the metrics serving.
 	MetricsBindAddress string
 
+	// MetricsSecure tells metrics server to use TLS configuration to serve
+	// provided metrics address.
+	// Defaults to false
+	MetricsSecure bool
```

Comment:
It seems at the moment, that providing a MetricsCertDir forces the metrics serving on, and that this only takes effect if the MetricsCertDir is empty? I wonder if it's worth just not having this and requiring the user to specify the directory and not defaulting it?